stable/spring-cloud-data-flow/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Toolkit for building data processing pipelines.
 name: spring-cloud-data-flow
-version: 2.5.1
+version: 2.6.0
```

Comment:
this would be `2.5.2` rather than `2.6.0`

Reply:
Fixed.